Container-License-Compliance/flows-annotations.md

```diff
@@ -0,0 +1,137 @@
+# Docker flows texts
+
+## (1) Dockerfile provenance
+
+Did you create your DockerFile from scratch, or did you reference or copy an existing DockerFile?
+
+If you referenced or copied an existing DockerFile, did that DockerFile container a licence? 
+
+Ensure you know the provenance of your DockerFile contents.
+
+Ensure you comply with any inbound license obligations.
+
+License your Dockerfile, especially if you are shipping it.
+
+
+## (2) FROM base layer/images
+
+Understand where your base image layers come from.
+
+Understand what base distro your base image is derived from.
+
+Confirm that your base layers are acceptable to the OSPO and security teams.
+
+Check that your base layers are ideally from Docker Official Images.
+
+Does your base layer come with any obligations, such as attributions or providing source code?
+
+
+## (3) Importing code from outside
+
+Did any of your FROM or COPY lines import code that originates outside our organization?
+
+Ensure you understand what code you imported, from where.
+
+Ensure you understand any license implications of that code.
+
+Ensure you list that code in your BOM if applicable.
+
+Ensure you list that code in your approval request if applicable.
+
+Ensure if you download code into your layers, you use a secure channel to download and/or check the validity of SHA/checksums if available.
+
+
+## (4) Importing code from inside
+
+Ensure you understand if all the imported code is from your project, or imported from other parts of intel as well.
+
+Ensure all of the code appears in your approval request slide deck, as appropriate.
```

Comment:
some organizations may not have the requirement to prepare a slide deck, some only may require a "document"